chapters/prg/python/python-install.md

```diff
@@ -384,16 +375,62 @@ Add the following line to your .bashrc, .bash_profile, or .zprofile
 source <path_to_conda_install>/etc/profile.d/conda.sh
 ```
 
+### Installing Python via `conda`
+
+Once you have installed conda, you can  install Python 3.10 in a virtual environment
+with conda please use 
+
+```bash
+$ conda create -n ENV3 -c conda-forge python=3.10.2 pip
```

Comment:
Note that this command does two things:

1. Creates a new environment named ENV3
2. Installs in that environment Python 3.10.2, and the latest version of pip that functions with python 3.10.2

This does not activate the environment, a user will need to run the `conda activate ENV3`, which I explain later in the document.